### DISCUSSION TITLE
New keybind for  SpitfireLFMkIX: 'Undercarriage UP else DOWN (2-way Switch)'

### DISCUSSION THREAD

**Discussion by correca** [Munkwolf/dcs-community-keybinds#28] [General]
Hi, this is the reverse action of the existing keybind 'Undercarriage DOWN else UP (2-way Switch)'

		-- Starboard Wall

{cockpit_device_id = devices.CONTROLS, down = device_commands.Button_25, up = device_commands.Button_25, value_down = 0, value_up = 1, name = _('Undercarriage UP else DOWN (2-way Switch)'), category = {_('Systems'), _('Custom')}},

**Comment by Munkwolf** [comment]
Thanks, this has been added along with some other reverse else commands.
